bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -208,7 +208,13 @@ <h2 class="primary go">{{ _('Teach the Web') }}</h2>
         <li id="promo-7" class="item promo-face">
           {{ promo_face(3, 'promo-7') }}
         </li>
-      {% if l10n_has_tag('webmaker_promo_thimble') %}
+      {% if l10n_has_tag('smarton_promo_tracking') and waffle.switch('tracking-protection') %}
+        <li id="promo-8" class="item promo-small-landscape smarton-tracking" tabindex="0">
+          <a class="panel-link" rel="external" href="#TODO-url('teach.smarton.tracking')" data-promotion-id="promo-8" data-promotion-name="Get smart on Tracking" data-promotion-class="promo-small-landscape" data-promotion-type="tile">
```

Comment:
Note to self: this link needs to be updated when #3443 merges (and this branch gets rebased)